pkg/domain/entities/images.go

```diff
@@ -47,6 +49,32 @@ func (i *Image) Id() string {
 	return i.ID
 }
 
+type ImageData struct {
```

Comment:
Duplicating this seems like a bad idea; we should move the backend structs into libpod/define and make this an alias for that type, as we did for container inspect.